kamulegeya/BankMgt
# Fix/not null bug and try to reformat the code

### Changes
@kamulegeya  I  ended up making a bunch of changes, we can revert them if the code does not make sense to the business logic

I also think you don't need the `Requisition_total` model, I implemented some else to track the totals. 

If this PR is ok you can merge it to master.

###  how to test locally
- This should work locally out of the box, no need to run any migrations, 
- if you want to test only what I have worked on locally,  checkout to this branch `fix/not-null-bug`  after pulling this branch
```bash
$ python manage.py runserver 
```
